docs/en/04_Changelogs/4.11.0.md

```diff
@@ -89,6 +92,19 @@ SilverStripe\CMS\Model\SiteTree:
 New installations using `silverstripe/silverstripe-installer` and `silverstripe/recipe-kitchen-sink` from 4.11 onwards will have the above plugins added to the `allowed-plugins` configuration by default.
 - From July 2022 composer will no longer prompt to allow plugins when running `composer install`. This won't affect new installs using silverstripe/installer or silverstripe-recipe-kitchen-sink, but will affect other new projects, and existing projects where `allowed-plugins` hasn't yet been defined. In those cases developers will need to declare the allowed plugins manually in the project's `composer.json` file.
 
+### Users will recieve an email if their password is changed {#change-password-email}
+
+The `SilverStripe\Security\Member.notify_password_change` configuration has been set to `true` by default - it used to be `false`. This means when a user changes their password on a project in "live" mode, they will recieve an email alerting them that their password was changed. The email includes a link to change their password again, so that users can recover their account in the event that someone else changed their password without their knowledge.
```

Comment:
Mention that sites with the `cwp/cwp-core` module installed already will already have the `notify_password_change` set the `true`

Reply:
Done